jdk_8_maven/em/embedded/rest/ocvn/src/main/java/em/embedded/org/devgateway/ocvn/EmbeddedEvoMasterController.java

```diff
@@ -146,7 +146,12 @@ public void resetStateOfSUT() {
 
     @Override
     public List<AuthenticationDto> getInfoForAuthentication() {
-        return Arrays.asList(AuthUtils.getForDefaultSpringFormLogin("ADMIN", "admin", "admin"));
+
+        // Adding a new user called user1 with password "password". User details are stored in the database.
```

Comment:
this comment is something would make sense on Git, not here.

Reply:
Hi Andrea, thanks for your comment. I fixed that.